ecl2df/compdat.py

```diff
@@ -155,6 +157,9 @@ def deck2dfs(deck, start_date=None, unroll=True):
     if unroll and not welsegs_df.empty:
         welsegs_df = unrolldf(welsegs_df, "SEGMENT1", "SEGMENT2")
 
+    if "KEYWORD_ID" in compdat_df.columns:
```

Comment:
Should we drop it, or could it become handy to keep for some other use later?

Reply:
(who knows what problems will creep  up when we finally manage to auto-merge these three dataframes)

Reply:
thinking of issue #6

Reply:
It will definitely come in handy - for the same reasons. But I'd say we can mention that in that issue (extend usage of `KEYWORD_IDX`). If I would not return `KEYWORD_IDX` I'd most likely break many other things (at least some tests).

Reply:
ok!